### DISCUSSION TITLE
.png file not recognized

### DISCUSSION THREAD

**Discussion by mveccia** [GrandOrgue/grandorgue#1808] [Developers]
Throughout the odf, .png are recognized, but if I insert an enclosure I get this error:
Error: bitmap format does not match for 'Enclosure001'
the code is
Bitmap001=Data\Positive\Espressiv\Swell-0.png
Bitmap002=Data\Positive\Espressiv\Swell-1.png
Bitmap003=Data\Positive\Espressiv\Swell-2.png
....
and .png are where they belong...!!!

**Comment by larspalo** [comment]
Are all the .png files of exactly the same size (resolution)? If size differs with even only one pixel it will cause an error.
